### PR TITLE
Update for Powerview Gen3 support

### PR DESCRIPTION
**Pull request by kingy444**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108066
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/hunterdouglas_powerview.markdown [hunk 1]
```diff
@@ -38,39 +38,57 @@ There is currently support for the following device types within Home Assistant:
...
 </div>
 
-{% include integrations/config_flow.md %}
+### Generation 3
 
-## Shades
+Generation 3 introduced RF Radio Pebble Remotes.
 
-### Known working devices
+Generation 3 shades report position changes back to the hub automatically and should not require any additional automations or considerations for positioning to appear correctly in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/hunterdouglas_powerview.markdown]:
```suggestion
Generation 3 shades report position changes back to the hub automatically. They should appear correctly in Home Assistant without requiring any additional automations or considerations for positioning.
```

File: source/_integrations/hunterdouglas_powerview.markdown [hunk 2]
```diff
@@ -38,39 +38,57 @@ There is currently support for the following device types within Home Assistant:
...
+
+## Shades
 
 <div class="note">
-If your shade is not listed please raise a feature request on the community forum.
+Your shades may still make work even if not listed. If you encounter issues please raise a feature request on the community forum.
```
Comment (c0ffeeca7) [inline source/_integrations/hunterdouglas_powerview.markdown]:
```suggestion
Your shades may still make work even if not listed. If you encounter issues, please raise a feature request on the community forum.
```

File: source/_integrations/hunterdouglas_powerview.markdown [hunk 3]
```diff
@@ -38,39 +38,57 @@ There is currently support for the following device types within Home Assistant:
 - Select
 - Sensor
 
+{% include integrations/config_flow.md %}
+
+## Hub Capabilities
```
Comment (c0ffeeca7) [inline source/_integrations/hunterdouglas_powerview.markdown]:
```suggestion
## Hub capabilities
```
I realize heading style caps are used elsewhere on this page. But in general we use sentence-style capitalization also in headings.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @kingy444 :+1:

**Comment by bdraco** [comment]
The parent merged into an integration branch and hasn't merged to dev yet. We still need a PR to merge it to dev

**Comment by frenck** [comment]
Meh @bdraco that PR was linked as the parent...  What to do at this point? Just revert?

**Comment by bdraco** [comment]
I think the integration branch is ready to merge to dev so probably doesn't matter so much as we have a month to get it sorted

**Comment by frenck** [comment]
I'll mark it and will monitor 👍

**Pull request merged**